src/assets/icons/bt_add_to_cart.svg

```diff
@@ -0,0 +1,6 @@
+<svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
los SVG te recomiendo pasarlos por https://jakearchibald.github.io/svgomg/
en el caso de este tienes una reducción de peso del 40%

<img width="1092" alt="Screenshot 2022-01-24 at 08 07 14" src="https://user-images.githubusercontent.com/45683136/150737241-c9a39881-f9e5-4985-a99a-ec2cfcdd8c65.png">

Reply:
I'm going to do that right now. And thanks for this page. I knew TinnyPNG but I didn't know that exist a wepage to do the same for the svg files.